docs/prerequisites/dependencies.md

```diff
@@ -0,0 +1,51 @@
+# Dependencies
+
+Before [executing](../execution/execution.md) ScubaGear, its dependencies must be installed:
+
+```powershell
+# Install the minimum required dependencies
+Initialize-SCuBA 
+```
+
+`Initialize-SCuBA` will install several modules on your system. These modules are listed in [RequiredVersions.ps1](https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/RequiredVersions.ps1).  It will also install [OPA](https://www.openpolicyagent.org).
+
+> **Note**: The `Initialize-SCuBA` cmdlet creates a `.scubagear` folder in the users home directory.  This is where it stores OPA and any other related files.
+
+## OPA Installation
+
+Normally, the `Initialize-SCuBA` cmdlet installs OPA.  This can be verified by looking for the OPA executable file in `C:\Users\<username>\.scubagear\Tools`.  If it failed to do so, or you set a parameter to prevent it from doing so, you can install OPA separately.
+
+Try this cmdlet first:
+
+```powershell
+# Download OPA
+Install-OPA
+```
+
+If that fails, you can manually download the OPA executable.
+
+* Go to [OPA download site](https://www.openpolicyagent.org/docs/latest/#running-opa).
+
+* In the upper left corner, select a version of OPA that is compatible with ScubaGear.
+
+![version](../images/opa_version.png)
+
+> **Note**: To find the default supported version, go to the [ScubaConfig](https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/Modules/ScubaConfig/ScubaConfig.psm1) file, look for the variable `$ScubaDefaults`, and find its parameter `DefaultOPAVersion`.  
+
+> **Note**: To find older supported versions, go to the [Support](https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/Modules/Support/Support.psm1) file, and find the constant named `$ACCEPTABLEVERSIONS`.
```

Comment:
It's unclear to me if having ScubaGear itself emit these dependencies is actually a good idea, because it requires a functional, updated ScubaGear to learn what is required to install it.  Rather a chicken and an egg problem.  I think putting this in the config file (see previous comment) is a better idea.